tests/test_models.py

```diff
@@ -437,7 +437,7 @@ def test_copy_brief(self, live_dos_framework):
         assert copy.framework == self.brief.framework
         assert copy.lot == self.brief.lot
         assert copy.users == self.brief.users
-        assert copy.copied_from_brief_id == self.brief.id
+        assert copy.is_a_copy is True
```

Comment:
Do we also want a test somewhere that `is_a_copy is False` for a non-copied brief?  Could probably tag an assertion into an existing test that creates a brief?